app/views/browse/build.html

```diff
@@ -20,14 +20,14 @@
                       data-original-title="Building from build configuration {{buildConfig.metadata.name}} has been paused.">
                 </span>
                 <small class="meta">created <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></small>
-                <div class="pull-right dropdown">
+                <div class="pull-right dropdown" ng-hide="!('builds' | canIDoAny)">
                   <!-- Primary Actions -->
                   <button class="btn btn-default hidden-xs"
                           ng-click="cancelBuild()"
-                          ng-if="!build.metadata.deletionTimestamp && (build | isIncompleteBuild)">Cancel Build</button>
+                          ng-if="!build.metadata.deletionTimestamp && (build | isIncompleteBuild) && ('builds/clone' | canI : 'create')">Cancel Build</button>
```

Comment:
create on builds/clone isnt the right check for cancel build, copy/paste error i'm guessing, same thing on line 42 below.  Should just be checking update on builds i think.